components/DocsHelp.tsx

```diff
@@ -250,7 +250,7 @@ export function DocsHelp() {
               target='_blank'
               rel='noreferrer'
               className='px-[16px] py-[8px] cursor-pointer border-solid border-[#aaaaaa] border rounded-md hover:bg-gray-200'
-              href='https://github.com/orgs/json-schema-org/projects/16'
+              href='https://github.com/json-schema-org/website'
```

Comment:
This should be a static link to the repo.  Instead this should be a link to the page you are actually visiting.

Reply:
[Uploading Screencast from 2024-03-12 10-42-57.webm…]()
Done.